ctapipe/containers.py

```diff
@@ -570,6 +571,20 @@ class TelescopeImpactParameterContainer(Container):
     distance_uncert = Field(nan * u.m, "uncertainty in impact_parameter", unit=u.m)
 
 
+class SimulatedPixelMonitoring(Container):
+    """
+    True information about nsb rate, quantum efficiency, high voltage, current and amplification settings for each pixel and telescope.
+    """
+    nsb_rate = Field(np.int32(-1), "NSB rate in units of p.e. per ns"
+            )
+
+    qe_rel = Field(None, "Quantum efficiency of photon detectors"
+            )
+
+    hv_rel = Field(None, "High voltage")
```

Comment:
might want to add in the help string something about why this is "rel".  Does that mean relative? relative to what?

Reply:
It's relative to the nominal high voltage, which should be in the camera configuration object.

Same for the `qe_rel`.

Maybe it makes sense to remove the `rel_` here completely and multiply with the value from the initial settings?

Reply:
```
   double nsb_pe_rate[H_MAX_PIX];  ///< NSB pixel p.e. rate [p.e./ns]
   double qe_rel[H_MAX_PIX];       ///< Assumed QE/PDE w.r.t. nominal
   double gain_rel[H_MAX_PIX];     ///< Assumed (PMT/common) gain w.r.t. nominal
   double hv_rel[H_MAX_PIX];       ///< Assumed high voltage w.r.t. nominal
   double current[H_MAX_PIX];      ///< Assumed current at pixel 
   double fadc_amp[H_MAX_GAINS][H_MAX_PIX]; ///< Assumed FADC amplitude per mean p.e.
   BYTE disabled[H_MAX_PIX];       ///< Pixel totally off and/or disabled in trigger.
```

Reply:
> Maybe it makes sense to remove the rel_ here completely and multiply with the value from the initial settings?

Yes, I think having it be the absolute HV is more clear - the nominal doesn't really matter in practice.